### PR TITLE
codeql-analysis.yml

### PR DESCRIPTION
**Pull request by MegoyTambayan**

### CODE REVIEW

**Comment by minimalsm** [comment]
Hey @MegoyTambayan ✌️ Thanks for the PR.

Can you provide a little context around what this is and why you've added it?

@wackerow @samajammin probably on you to review

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        # Prefix the list here with "+" to use these queries and those in the config file.
+        # queries: ./path/to/local/query, your-org/your-repo/queries@main
+
+    # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
+    # If this step fails, then you should remove it and run the build manually (see below)
+    - name: Autobuild
+      uses: github/codeql-action/autobuild@v1
+
+    # ℹ️ Command-line programs to run using the OS shell.
+    # 📚 https://git.io/JvXDl
```
Comment (jley81) [inline .github/workflows/codeql-analysis.yml:60]: What is this?

Comment (jley81) [reply .github/workflows/codeql-analysis.yml:60]: It's me fool.

**Review by jley81 — APPROVED** [review]

**Review by MegoyTambayan — COMMENTED** [review]
Hmm

**Comment by minimalsm** [comment]
Hey @MegoyTambayan. Thanks again for the PR 🙂. We're not really sure what this is for. Doing some housekeeping so I'm going to close it out for now but feel free to reopen with additional information. ✌️

**Pull request closed without merging**